transcom/mymove
# B 19586 int

## [B-19586](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A943214)

## Summary

When shipments are approved, the move  is made available to the prime. During the process of making the move available to the prime, the system also adds the Move Management service item. This service item should only exist once per move, so a condition was added.

Is there anything you would like reviewers to give additional scrutiny?


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Be sure to pay special attention to step 15

1. Access the client application
2. Login as a new user
3. Set your user details
4. Create a move with more than one shipment
5. Submit the move details
6. Log out of the client application
7. Access the Office application
8. Log in as a Service Counselor
9. Find the move you created
10. Submit the move details
11. Log out of the Office application
12. Access the Office application
13. Log in as a TOO
14. Find the move you created
15. Select ONLY ONE of the shipments (if you approve them all at once you will have to restart)
16. Approve the selected shipment
17. Navigate to the Move details
18. Verify that the Move Management service item exists
19. Select another shipment 
20. Approve the selected shipment
21. Navigate to move details
22. Verify that the Move Management service item is only present one time

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
